test/specs.test.js

```diff
@@ -11,7 +11,9 @@ const clone = require('clone');
 
 const dir = path.resolve(__dirname, 'specs'),
     files = fs.readdirSync(dir).filter(function (f) {
-        return f.match(/\.yaml$/);
+        const isYaml = f.match(/\.yaml$/);
+        const grep = process.env.GREP ? (new RegExp(process.env.GREP)).test(f) : true;
```

Comment:
This is built into mocha. Just do `npm test -- -- grep functions` and you will be good to go. 👍